testdata/baselines/reference/submodule/compiler/declarationEmitExpandoPropertyPrivateName.js.diff

```diff
@@ -14,4 +14,7 @@
  export declare function f(): I;
  export {};
 +//// [b.d.ts]
```

Comment:
It's super odd that the original code did not emit a declaration file here, not sure why (the new code looks correct).